Assets/Scripts/FMOD/ScriptMusicTimeline.cs

```diff
@@ -66,7 +66,7 @@ class TimelineInfo
 
         static bool beatTrigger = false;
         static float beatWindowAfter;
-
+        public bool onTempo = false;
```

Comment:
Use a bunch of observers instead that listen to the beat.
```suggestion
        public List<ISyncable> syncable = new()
```

Reply:
I would prefer doing this. I would also prefer to have this when I do the tempo refactor